docs/handbook/writing-your-own-file-decoder.rst

```diff
@@ -103,6 +105,13 @@ Note that the image plugin must be explicitly registered using
 :py:func:`PIL.Image.register_open`. Although not required, it is also a good
 idea to register any extensions used by this format.
 
+Once the plugin has been imported, it can be used::
+
+    from PIL import Image
```

Comment:
Shall we add explicit Python code blocks in this file too?

```suggestion
Once the plugin has been imported, it can be used:

.. code-block:: python

    from PIL import Image
```

Looks like things are already autocoloured for Python at https://pillow.readthedocs.io/en/stable/handbook/writing-your-own-file-decoder.html but being explicit will also colour it here on GitHub: https://github.com/python-pillow/Pillow/blob/master/docs/handbook/writing-your-own-file-decoder.rst